docs: add REWE International to ADOPTERS.md (#5335)
cloudnative-pg/cloudnative-pg: ADOPTERS.md

| [Google Cloud](https://cloud.google.com/) | @mastersingh24 | 2024-03-12 | Leverage the full potential of cutting-edge PostgreSQL  and CloudNativePG  on [Google Kubernetes Engine (GKE)](https://cloud.google.com/kubernetes-engine) with EDB Community 360 PostgreSQL available in the [Google Cloud Marketplace](https://console.cloud.google.com/marketplace/product/public-edb-ppas/edb-postgresql). |
| [Syself](https://syself.com) | @batistein | 2024-05-06 | Syself offers a simplified, multi-cloud Managed Kubernetes platform based on Cluster API and uses CloudNativePG for managing Postgres clusters in our internal infrastructure. |
| [ParadeDB](https://paradedb.com) | @philippemnoel | 2024-07-10 | ParadeDB is an Elasticsearch alternative on Postgres. It leverages CloudNativePG to manage ParadeDB Postgres clusters which connect to a customer's existing Postgres infrastructure via logical (streaming) replication. |

> | [REWE International AG](https://rewe-group.at/de) | @rewemkris | 2024-08-21 |Hello ! 👋 We are the DBMS Team of the RIAG IT and are responsible for managing databases around the world for our stores, warehouses and online shops. We´re leveraging CloudNativePG's Operator by providing PostgreSQL as a Service and creating highly available databases running on Kubernetes in Google Cloud & OnPremise. |

Reviewer: ```suggestion
| [REWE International AG](https://rewe-group.at/en) | @rewemkris | 2024-08-21 |Hello! 👋 We are the DBMS Team of RIAG IT, responsible for managing databases worldwide for our stores, warehouses, and online shops. We leverage CloudNativePG to provide PostgreSQL as a Service, creating highly available databases running on Kubernetes in both Google Cloud and on-premises environments.|
```
